PIL/Image.py

```diff
@@ -1784,6 +1862,7 @@ def rotate(self, angle, resample=NEAREST, expand=0, center=None,
         ]
 
         def transform(x, y, matrix):
+            # type: (float, float, Sequence[float]) -> Tuple[float, float]
```

Comment:
a) The code uses a `List` :)
b) I tested with 2.7 and 3.5, and `n=[3,4];(c,d)=n;c;d` gives what you'd hope for.
c) This is only used within `rotate`, so if it passes mypy we're probably good, in terms of defining a public interface at least.